docs/en/development/solution-templates.md

```diff
@@ -0,0 +1,34 @@
+# Solution Templates
+
+Save time by using one of our templated solutions! We're leveraging `dotnet new`([see Microsoft developer docs](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-new)) here.
+
+## The Portal Site Template
+
+* This templates sets up a new solution that uses the `Portal Theme`
```

Comment:
Maybe we should include a starter theme with basic assets as part of the Template. 

Why did you use "Portal Site Template" here ?

Reply:
Doesn't this template set up for web portals?

Reply:
It's set up for building modules / themes using our packages by default. Not for portal in particular